src/libm-tester/tester.c

```diff
@@ -4311,7 +4316,7 @@ void do_test() {
 
     //
 
-    mpfr_set_default_prec(1280);
+    mpfr_set_default_prec(320);
```

Comment:
Did you reduce the precision of MPFR because it gives speedup? Why did you choose the original values? Are these new values enough to make sure we don't miss errors?

Reply:
1280 is for double precision.
320 is more than 128(exponent of single precision) + 24(mantissa length). Actually, 160 should be enough.